src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -15,11 +15,14 @@
       import org.apache.commons.cli.Option;
       import org.apache.commons.cli.Options;
     @end
-    // [START {@sample.regionTag}]
     @if sample.sampleImports.appImports.size
       {@importList(sample.sampleImports.appImports)}
 
     @end
+    @if sample.sampleImports.appImports.size
```

Comment:
Don't you want this inside the region tag? See comment in baseline.

Reply:
My bad! Will fix it soon.